defaults/main.yml

```diff
@@ -1,4 +1,7 @@
 ---
+pgbouncer_user: "{{ 'pgbouncer' if ansible_os_family == 'RedHat' else 'postgres' }}"
```

Comment:
Previously, the username was loaded from `vars/Debian.yml`
Why did not it work?

Reply:
Short answer. This changed because of variable precedence. defining `pgbouncer_user` and `pgbouncer_group` in defaults/main.yml allowed me to overwrite from `group_vars` or `var_files`

Long answer. In order to set the proper ownership on the certificate/key files in [miarec/ansible-miarec/prepare-hosts.yml](https://github.com/miarec/ansible-miarec/blob/20230825-redis_tls/prepare-hosts.yml) and [miarecdevops/miarec-all-in-one-aws/ansible/prepare-hosts.yml](https://github.com/miarecdevops/miarec-all-in-one-aws/blob/20231004-db-tls/ansible/prepare-hosts.yml). I defined variable `pgbouncer_user` and `pgbouncer_group` using the following methods;
 - play `var_files` (miarec/ansible-miarec/vars/db.yml) 
 - `group_vars` (miarecdevops/miarec-all-in-one-aws/ansible/group_vars/db.yml).  

These methods have a lower precedence than the `include_vars` method used in the role [miarec/ansible-role-pgbouncer/tasks/include-vars.yml](https://github.com/miarec/ansible-role-pgbouncer/blob/master/tasks/include-vars.yml).    

Some alternative: 
 - setting the ownership of the certificate/key in the role, but that would be inconsistent with the postgresql and redis roles.
 - using `set_fact` to define the variable in the playbook. This would work also.